src/containers/HomeHeader/HomeHeader.tsx

```diff
@@ -55,6 +47,14 @@ export default function HomeHeader({
                       </p>
                   </>
                 }
+                { (unrealizedPnl !== null && totalSpent !== null) &&
+                  <>
+                      <label>{trans('home.title.summary.pnl')}</label>
+                      <p className={unrealizedPnl.gt("0") ? s.link__green : s.link__red }>
+                          {unrealizedPnl.toString()}% / {totalSpent.charAt(0)}{ (Number(totalSpent.slice(1)) + ((Number(unrealizedPnl)/100) * Number(totalSpent.slice(1)))).toFixed(2) }
```

Comment:
I don't wanna nitpick (sorry) But it's better to create an object separating the value and the symbol and store that in redux. It took me a bit to understand what was going on here.